cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/cache.go

```diff
@@ -160,6 +162,33 @@ func newCacheWithRedisClient(config cacheConfig, redisCli *redis.Client, waitGro
 	return &c, nil
 }
 
+func (c NamespacedResourceWatcherCache) isGvrValid() error {
+	if c.config.gvr.Empty() {
+		return status.Errorf(codes.FailedPrecondition, "server configured with empty GVR")
```

Comment:
Not sure why grpc status errors are used in this function... the call-site isn't returning a response I don't think? If it's just on startup, normal errors are fine (though I don't see any issue using grpc ones).

Reply:
it had something to do with unit tests, I think. Not important enough now to go into. Will change if/when needed